internal/mvcc/kv.go

```diff
@@ -104,6 +104,8 @@ type KV interface {
 	// Read creates a read transaction.
 	Read() TxnRead
 
+	StaleConcurrentRead() TxnRead
```

Comment:
let the implementation behind `Read()` handle the stale fallback instead of exposing it?